examples/muscle_driven_with_contact/2segments_4dof_2contacts_1muscle.bioMod

```diff
@@ -0,0 +1,78 @@
+version 4
+
```

Comment:
Is that usefull to have duplicated this file instead of just adding muscle to the original file?